scss/styles.scss

```diff
@@ -1,22 +1,22 @@
-// Libraries 
-@import "lib/bourbon/_bourbon.scss";
-@import "lib/neat/_neat.scss";
-@import "fonts";
+// Libraries
+@import 'lib/bourbon/_bourbon.scss';
```

Comment:
Imported partial `lib/bourbon/_bourbon.scss` should be written as `lib/bourbon/bourbon`